suites/pacific/cephadm/tier1_cephadm_bootstrap.yaml

```diff
@@ -0,0 +1,141 @@
+      desc: MGR,MON,OSD,RGW,MDS,HOST,CLIENT deployment.
+      module: test_cephadm.py
+      polarion-id:
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  nodes:
+                      - node1
+                      - node2
+                      - node6
+                  sep: ";"
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+          - config:
+              command: shell
+              args:                             # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply
+              service: mds
+              base_cmd_args:                    # arguments to ceph orch
+                verbose: true
+              pos_args:
+                - cephfs                        # name of the filesystem
+              args:
+                placement:
+                  nodes:
+                    - node2
+                    - node8
+          - config:
+              command: apply
+              service: rgw
+              base_cmd_args:                    # arguments to ceph orch
+                verbose: true
+              pos_args:                         # positional arguments
+                - india                         # realm
+                - south                         # zone
+              args:
+                placement:
+                  label: rgw                    # label
+              get_cluster_details:              # to view ceph status only
+                - "ceph status"
+                - "ceph orch ls rgw -f json-pretty"
+                - "ceph orch ps '' rgw.india.south -f json-pretty"
+                - "ceph health detail -f yaml"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Configure client
+      desc: Configure client on node9
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node9                       # client node
+        install_packages:
+          - ceph-common                   # install ceph common packages
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
```

Comment:
capabilities is for another new client entry, where this would be used for performing IO operation maybe like(Kernel, fuse mount .,etc), in-case admin is not used.